internal/custom_executable/main.c.txt

```diff
@@ -5,20 +5,22 @@
 int main(int argc, char *argv[]) {
     if (argc != 2) {
         if (argc < 2) {
-            printf("Expected exactly one command line argument, got 0\n");
+            printf("%s: Expected exactly one command line argument, got 0\n", argv[0]);
```

Comment:
@ryan-gang let's remove these error conditions entirely - they'll cause confusing error messages. Let's just print out the format below (iterate over args)